oaknational/Oak-Web-Application
# feat: transcripts on cms videos - LESQ-784

## Description

Music year: 1978

- Adds captions to CMS video schema, GQL and component
- Refactors lesson overview transcript viewer
- Adds transcript toggle component 



## How to test

1. Go to https://deploy-preview-2421--oak-web-application.netlify.thenational.academy
Videos will need to be re-published on Sanity studio to see transcripts
- plan a lesson - https://deploy-preview-2421--oak-web-application.netlify.thenational.academy/lesson-planning
- about us - who we are - https://deploy-preview-2421--oak-web-application.netlify.thenational.academy/about-us/who-we-are
- webinar and blogs - https://deploy-preview-2421--oak-web-application.netlify.thenational.academy/webinars/market-engagement-webinar-july-2023
https://deploy-preview-2421--oak-web-application.netlify.thenational.academy/blog/how-to-design-a-subject-curriculum



## Checklist
